frontend/javascripts/admin/admin_rest_api.js

```diff
@@ -981,16 +981,10 @@ export async function getOrganizationForDataset(datasetName: string): Promise<st
 }
 
 export async function findDataPositionForLayer(
-  datastoreUrl: string,
-  datasetId: APIDatasetId,
-  layerName: string,
+  requestUrl: string,
 ): Promise<{ position: ?Vector3, resolution: ?Vector3 }> {
   const { position, resolution } = await doWithToken(token =>
-    Request.receiveJSON(
-      `${datastoreUrl}/data/datasets/${datasetId.owningOrganization}/${
-        datasetId.name
-      }/layers/${layerName}/findData?token=${token}`,
-    ),
+    Request.receiveJSON(`${requestUrl}/findData?token=${token}`),
```

Comment:
I think, this function becomes hard to use when only requiring a `requestUrl` since it's not clear how that has to be constructed. I assume you changed this since the requirements for data layers vs. volume tracings differ too much, which is why I suggest to have two API functions here: `findDataPositionForLayer` and `findDataPositionForVolumeTracing`.